File: problems/easy/savings/__tests__/savings.js
const count_savings = require("../savings");

let  testnum = [111,175,999,1,925];
let expected = [6216,15400,499500,1,428275];
for (let index = 0; index < testnum.length; index++) {
    if (count_savings(testnum[index])!=expected[index]) {
        throw new Error(`❌ Test failed. Expected YES, got ${result}`);
        break;
    }
    
}

console.log("✅ JavaScript test passed!");

File: problems/easy/savings/__tests__/savings.py
import sys
import os

sys.path.insert(0, os.path.abspath(os.path.join(os.path.dirname(__file__), '..')))

from savings.savings import count_savings

testnum = [111,175,999,1,925]
expected = [6216,15400,499500,1,428275]

def testSavings():
    for x in range(len(testSavings)):
        if count_savings(testSavings[x]) != expected[x]:
            return False
            break
    return True

if __name__ == "__main__":
    if testSavings() == True:
        print("✅ Python test passed!")

    else: print("❌ python test failed")